src/avram/migrator/alter_table_statement.cr

```diff
@@ -6,10 +6,28 @@ class Avram::Migrator::AlterTableStatement
   getter rows = [] of String
   getter dropped_rows = [] of String
   getter fill_existing_with_statements = [] of String
+  getter change_type_statements = [] of String
 
   def initialize(@table_name : Symbol)
   end
 
+  macro change_type(type_declaration, **type_options)
+    {% if !type_declaration.is_a?(TypeDeclaration) %}
+      {% type_declaration.raise "Must pass a type declaration to 'change_type'. Example: change_type age : Int32" %}
+    {% end %}
+    %column = ::Avram::Migrator::Columns::{{ type_declaration.type }}Column({{ type_declaration.type }}).new(
```

Comment:
Yeah it would have taken me weeks to figure this one out

Reply:
I’ll try to see if I can make this type of change easier. I’ve got some ideas I’ll try out later!